articles/anomaly-detection/concepts/credential-stuffing.md

```diff
@@ -0,0 +1,43 @@
+---
+description: Understand how Auth0 detects anomalies to stop malicious attempts to access your application, alert you and your users of suspicious activity, and block further login attempts. 
+topics:
+    - security
+    - anomaly-detection
+    - credential-stuffing
+contentType: concept
+useCase: anomaly-detection
+---
+# Automated Attack and Credential Stuffing Protection
+
+Credential stuffing attacks (or *list validation attacks*) occur when bad actors automate the process of trying username and password combinations (usually stolen from another site) for many accounts in a short period of time.  According to recent statistics, as many as 71% of accounts use the same password across multiple sites so a credential stuffing attack has the potential to successfully log into your system.  
+
+Use **Automated Attack Protection** to provide a standard level of protection against credential stuffing attacks that does not add any friction to legitimate users. This protection is enabled by default for all connections.  
```

Comment:
I'd be careful with "does not add any friction to legitimate users", as we WILL have false positives and do things like throw captcha for a small number of legit users.

```suggestion
Use **Automated Attack Protection** to provide a standard level of protection against credential stuffing attacks with minimal friction for legitimate users. This protection is enabled by default for all connections.  
```